test/extended/networking/egress_firewall.go

```diff
@@ -128,6 +191,17 @@ func sendEgressFwTraffic(f *e2e.Framework, oc *exutil.CLI, pod string, nodeSelec
 	_, err = oc.Run("exec").Args(pod, "--", "curl", "-q", "-s", "-I", "-m3", "https://docs.openshift.com").Output()
 	expectNoError(err)
 
+	// TODO:
+	// - Check for coredns-ocp-dnsnameresolver plugin.
+	// - Remove TecHPreview check when feature is GA.
+	if networkPluginName() == OVNKubernetesPluginName && exutil.IsTechPreviewNoUpgrade(oc) {
+		// Test curl to registry.build05.ci.openshift.org should pass
+		// because we have allow dns rule for *.ci.openshift.org
```

Comment:
that comment doesn't look right

Reply:
Missed correcting comment when addressing a review comment.
Code comment updated.